analytics_data_api/v0/serializers.py

```diff
@@ -329,35 +332,47 @@ class Meta(object):
         )
 
 
+# pylint: disable=abstract-method
 class LastUpdatedSerializer(serializers.Serializer):
-    last_updated = serializers.DateField(source='date', format=settings.DATE_FORMAT)
+    last_updated = serializers.DateTimeField(source='date', format=settings.DATE_FORMAT)
```

Comment:
Ok, I think I get it. So we store it as a datetime, but only ever display/serialize the date part?